### ISSUE TITLE
mcp display no numbers in windows. 

### ISSUE THREAD

**Issue by steve-jacko444** [invernyx/md11-bugs#1136]
### Describe the bug

i have just run the last update, and there are no figures in the MCP windows 

### Expected behaviour

should have alt, hdg speed figures. 
![Picture1](https://github.com/invernyx/md11-bugs/assets/156470813/2da1b52c-f9dd-4746-a121-192444f6c4b3)

### Simulator

P3Dv5

### Crash

Yes

### Agreement

- [X] Yes

**Comment by steve-jacko444** [comment]
it didn't crash.

**Comment by steve-jacko444** [comment]
So I repeated this tonight, following advice to amend the cfg files, the no's show up after cycling states, so this is still not correct. 
The airplane will not follow FMS speed settings. 
setting the flaps causes the auto throttle to reduce thrust to meet flap speeds. 
landing light, still illuminate the cockpit on raising and lowering in non day light conditions. 
At this stage, I really though these issues would have been fixed. we have taken a backward step.
